### DISCUSSION TITLE
Getting Chromedriver error when running the tests in my local mobile device farm with different chrome versions (Android)

### DISCUSSION THREAD

**Discussion by RashmiShashindra** [webdriverio/webdriverio#10800] [Q&A] [answered]
`[1-0] 2023-07-25T06:52:09.510Z ERROR webdriver: Request failed with status 500 due to unknown error: An unknown server-side error occurred while processing the command. Original error: No Chromedriver found that can automate Chrome '103.0.5060'. See https://github.com/appium/appium/blob/master/docs/en/writing-running-appium/web/chromedriver.md for more details`

`[0-0] 2023-07-25T06:52:12.260Z ERROR webdriver: Request failed with status 500 due to unknown error: An unknown server-side error occurred while processing the command. Original error: No Chromedriver found that can automate Chrome '91.0.4472'. See https://github.com/appium/appium/blob/master/docs/en/writing-running-appium/web/chromedriver.md for more details`

`[2-0] 2023-07-25T06:52:35.230Z ERROR webdriver: Request failed with status 500 due to unknown error: An unknown server-side error occurred while processing the command. Original error: No Chromedriver found that can automate Chrome '114.0.5735'. See https://github.com/appium/appium/blob/master/docs/en/writing-running-appium/web/chromedriver.md for more details`

Getting these errors when the webview related tests are running. Please help!

Appium Log Attached: 
[appium.log](https://github.com/webdriverio/webdriverio/files/12158495/appium.log)

**Comment by christian-bromann** [comment]
Do you have Chromedriver installed?

**Reply by RashmiShashindra** [reply]
@christian-bromann Thanks for getting back. No, I have not installed the chromedriver manually. It comes with the webdriverio. These tests actually worked previously and suddenly started failing.

What I am trying to do in the test is to just switch to the Webview contexts. But it throws error that `No Chromedriver found that can automate Chrome '114.0.5735'.`

Tools & Technologies used:

- Webdriverio
- Appium ("@wdio/appium-service": "^7.19.5",)
- NodeJS

**Reply by RashmiShashindra** [reply, marked as answer]
@christian-bromann Found the issue reading the appium log. It was a permission issue in my local device farm 

/usr/local/lib/node_modules/appium/node_modules/appium-chromedriver/chromedriver/mac/

It only had Read permissions, where the appium could not download and include the drivers in the location. I changed the permissions now and it fixed the issue.

Thanks for looking into this.
